mission_control/navigator_missions/navigator_singleton/navigator.py

```diff
@@ -222,10 +221,10 @@ def catch_error(self, failure):
             # Handle error
 
     @util.cancellableInlineCallbacks
-    def start_search(self, timeout=60, loop=True, spotings_req=2, speed=0.1):
+    def start_search(self, timeout=60, loop=True, spotings_req=2, **kwargs):
```

Comment:
It gets used in that function:
`while self.nav.nh.get_time() - start_time < genpy.Duration(timeout):`